### DISCUSSION TITLE
Quarto visual editor in an akward "not quite yet WYSIWYG notebook" state

### DISCUSSION THREAD

**Discussion by MichelNivard** [posit-dev/positron#4612] [Feedback]
The quarto visual editor outputs in console not inline.  So, unlike in RStudio, its just a source editor with a bold button, not really a WYSIWWYG authoring solution where you can see your gt tables, figures, footnotes, layouts in the context of the doc while authoring.. I realise some or a lot of that functionality lives in .ipynb notebook and they are easily transformed from one to the other but that forces a cumbersome workflow on the most WYSIWYG part of the user base, which are quite like RStudio only ppl who dont have a prior affinity with or trust in VSCode. Could you make the visual editor up to par with the RStudio version, or allow .qmd files to be treated as proper computational notebooks? I realise that in console execution is really the VSCode way and will have its own fanbase, and I think the source editor should prob default to executing in console? I worry you'll leave the hardest to convince casual WYSIWYG coder clinging to RStudio.

**Comment by juliasilge** [comment]
Thanks for this feedback, which sounds to me more like it is wanting inline execution for Quarto documents (rather than the details of the visual editor itself). We know folks have strong feelings about this! As of today, what we recommend is:

- If you want an inline experience, use `.ipynb`
- If you want a send-to-console experience, use `.qmd`

If you want to check out other discussion we've had, you can take a look at https://github.com/posit-dev/positron/issues/1533, before/after https://github.com/posit-dev/positron/discussions/3718#discussioncomment-9938783, and https://github.com/posit-dev/positron/issues/1418.

We appreciate you sharing your preferences as we decide how to prioritize this! 🙌
